cli/shared/src/main/scala/com/lightbend/rp/reactivecli/runtime/kubernetes/PodTemplate.scala

```diff
@@ -359,9 +359,18 @@ object PodTemplate {
       else
         labels
 
+    val annotationsToUse = Option(
+      annotations
+        .annotations
+        .map(a => (a.key, jString(a.value)))
+        .toMap)
+      .filter(_.nonEmpty)
+
     PodTemplate(
       Json(
-        "metadata" -> Json("labels" -> labelsToUse.asJson),
+        "metadata" -> (
+          ("labels" -> labelsToUse.asJson) ->:
+          ("annotations" :=? annotationsToUse) ->?: jEmptyObject),
```

Comment:
I couldn't figure out an easier to read version of this in argonaut. It appears to be the way to handle exclusion of empty fields (see http://argonaut.io/doc/json/), but relies on knowledge of argonaut operators.